internal/graph/cached_resolver.go

```diff
@@ -150,37 +151,30 @@ func (c *CachedCheckResolver) ResolveCheck(
 ) (*ResolveCheckResponse, error) {
 	span := trace.SpanFromContext(ctx)
 
-	skipCache := c.enableConsistencyOptions && req.Consistency == openfgav1.ConsistencyPreference_HIGHER_CONSISTENCY
-
-	// if consistency options experimental flag is set, and consistency param set to HIGHER_CONSISTENCY, skip cache
-	if skipCache {
-		resp, err := c.delegate.ResolveCheck(ctx, req)
-		if err != nil {
-			telemetry.TraceError(span, err)
-			return nil, err
-		}
-		return resp, nil
-	}
-
-	checkCacheTotalCounter.Inc()
-
 	cacheKey, err := CheckRequestCacheKey(req)
 	if err != nil {
 		c.logger.Error("cache key computation failed with error", zap.Error(err))
 		telemetry.TraceError(span, err)
 		return nil, err
 	}
 
-	cachedResp := c.cache.Get(cacheKey)
-	isCached := cachedResp != nil && !cachedResp.Expired()
-	span.SetAttributes(attribute.Bool("is_cached", isCached))
-	if isCached {
-		checkCacheHitCounter.Inc()
+	skipCache := c.enableConsistencyOptions && req.Consistency == openfgav1.ConsistencyPreference_HIGHER_CONSISTENCY
```

Comment:
As an aside, we should have measurement tracking how frequently we have skip cache

Reply:
We have telemetry data that would allow us to count the total number of requests per consistency type (#1721), do you think we could use that to get the data you're thinking of?